app/web/src/components/RecipeItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import './RecipeItem.css';
+import { Recipe } from "../context/types";
+import { RecipeItemContainer, RecipeName, RecipeLabel } from "./Styles.tsx";
+
+const RecipeItem = (props: Recipe) => {
+  return (
+    <RecipeItemContainer>
+      <RecipeName>{props.name}</RecipeName>
+      <p>
+        <RecipeLabel>{props.description}</RecipeLabel>
+      </p>
+      {props.ingredients && (
+        <ul>
+          {props.ingredients.map((ingredient, index) =>
+            <li key={index}>
```

Comment:
If ingredient has an `id` attribute which is unique, it's better to use that instead of the index as a key

Reply:
I remember seeing something like that while studying react, but in this case I ended up using the index because there wasn't an ID. At least I didn't add one while building the backend.